ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/package.scala

```diff
@@ -22,4 +22,7 @@ package object ledger {
   val EventId: LedgerString.type = LedgerString
   type EventId = EventId.T
 
+  /** Identifiers used to correlate admin submission with results. */
+  val SubmissionId: LedgerString.type = LedgerString
```

Comment:
This should only be defined in the `participant-state` module.

Reply:
removed SubmissionId from these two modules 
![image](https://user-images.githubusercontent.com/25179017/69777864-12582f80-1170-11ea-9f79-d3f1816713e0.png)